src/Uno.UI/WasmCSS/Uno.UI.css

```diff
@@ -135,6 +135,7 @@ embed.uno-frameworkelement.uno-unarranged {
   background-clip: text !important;
   /* overflow:hidden is required for text-overflow: ellipsis to work correctly */
   overflow: hidden;
+  display: inline-flex; /* This is a workaround to prevent selection bleeding, see https://github.com/unoplatform/uno/issues/13440 */
```

Comment:
Does this have some potential disadvantage for us?

Reply:
We **do** nest children inside `p`. A TextBlock can have child `Run`s.